pkgs/tools/security/enum4linux/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, makeWrapper, samba, perl, openldap }:
+
+stdenv.mkDerivation rec {
+  pname = "enum4linux";
+  version = "0.8.9";
+  src = fetchurl {
+    url = "https://labs.portcullis.co.uk/download/enum4linux-${version}.tar.gz";
+    sha256 = "41334df0cb1ba82db9e3212981340372bb355a8160073331d2a1610908a62d85";
+  };
+
+  dontBuild = true;
+
+  buildInputs = [ makeWrapper samba perl openldap ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp enum4linux.pl $out/bin/
+
+    wrapProgram $out/bin/enum4linux.pl \
+      --prefix PATH : ${stdenv.lib.makeBinPath [ samba perl openldap ]}
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A tool for enumerating information from Windows and Samba systems";
+    homepage = "https://labs.portcullis.co.uk/tools/enum4linux/";
+    # https://github.com/portcullislabs/enum4linux/blob/master/COPYING.ENUM4LINUX
```

Comment:
Needed?

Reply:
I thought it might be interesting where the original license can be found. However, also fine for me to remove that comment :+1: